### ISSUE TITLE
IndexOutOfBoundsException and crash when opening inventory on 1.17

### ISSUE THREAD

**Issue by Archy-X** [MinusKube/SmartInvs#188]
The server throws an error and crashes when attempting to open a custom inventory on 1.17, this could be a spigot issue though:
https://pastebin.com/PtvKAAZD

**Comment by portlek** [comment]
could you show your code where you use the library or open the inventory.

**Comment by Archy-X** [comment]
It is just when calling SmartInventory#open, such as here:
```
SmartInventory inventory = SkillsMenu.getInventory(player, plugin);
if (inventory != null) {
	inventory.open(player);
} else {
	player.sendMessage(Lang.getMessage(CommandMessage.NO_PROFILE, Lang.getDefaultLanguage()));
}
```

**Comment by Archy-X** [comment]
I just updated to most recent version of SmartInvs, here is the new error: https://pastebin.com/qvapnmzR

**Comment by Archy-X** [comment]
Confirmed spigot issue and fixed on latest 1.17 build.
